lib/src/events/signal.dart

```diff
@@ -64,7 +64,20 @@ class Signal {
   }
 }
 
+enum EventSignalConfKey {
+  LongPressDuration,
+  LongPressShakingDistance,
+}
+
 class EventSignal<T> {
+  Map<EventSignalConfKey,Object> configure = {};
+  EventSignal();
+ factory   EventSignal.longPress(Duration duration,double distance) {
+    var signal = EventSignal<T>();
+    signal.configure[EventSignalConfKey.LongPressDuration] = duration;
```

Comment:
Another observation, even if the Duration type makes total sense in Dart/Flutter, GraphX is currently using double to express durations in the tween system and the "ticker" elapsed time since last frame ... ( 0.1 = 100 milliseconds). Just a thought, as that concept is taken from JS/AS3.

Reply:
I think Duration is better on user side,  maybe we can translate Duration to double ticks?